bin/install

```diff
@@ -7,7 +7,9 @@ if [[ "${ASDF_INSTALL_TYPE:-version}" == 'ref' ]]; then
   exit 1
 fi
 
-TARBALL_URL="https://registry.npmjs.org/pnpm/-/pnpm-${ASDF_INSTALL_VERSION}.tgz"
+REGISTRY=$(npm config get registry)
```

Comment:
This introduces a dependency on NodeJS/npm. Can we handle the case where that command fails, and default to registry.npmjs.org please